### DISCUSSION TITLE
How do I stop `k  Enter normal mode` appearing whenever I press `j`

### DISCUSSION THREAD

**Discussion by David-Else** [helix-editor/helix#2307] [Q&A]
I have:
```
[keys.insert]
j = { k = "normal_mode" }
```
Now when I hit a `j` the `k  Enter normal mode` message spams my status line. I don't want this to flicker on and off as I type. I think this should be a special case as it is a very common setting to have.

**Comment by the-mikedavis** [comment]
There's an issue for a configurable delay before the popup shows that IMO is the perfect solution for this: https://github.com/helix-editor/helix/issues/1475

Another approach could be to be able to configure it within the keymap, maybe something like

```toml
j = { k = "normal_mode", show_menu = false }
```

But meh, I think I like the delay approach better. It seems kinda hacky to me to add an extra special key to keymap tables.

**Reply by David-Else** [reply]
> There's an issue for a configurable delay before the popup shows that IMO is the perfect solution for this: #1475

That would be perfect! :)
